decompressors_test.go

```diff
@@ -0,0 +1,82 @@
+package fakemachine
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/go-debos/fakemachine/cpio"
+)
+
+func checkStreamsMatch(t *testing.T, output, check io.Reader) {
+	i := 0
+	oreader := bufio.NewReader(output)
+	creader := bufio.NewReader(check)
+	for {
+		ochar, oerr := oreader.ReadByte()
+		cchar, cerr := creader.ReadByte()
+		if oerr != nil || cerr != nil {
+			if oerr == io.EOF && cerr == io.EOF {
+				return;
```

Comment:
best to return an error in this function ?

Reply:
seems like this is just used for testing, so may not be worth refactoring

Reply:
Please let me know what you prefer - I'm fine either way.

Reply:
i'd rather errors were bubbled up - to follow the rest of the fakemachine/debos code ;-)

Reply:
Should be fixed with v2. Please let me know if you had something else in mind.